apps/embed-web/vite.config.ts

```diff
@@ -0,0 +1,36 @@
+import { resolve } from "path";
+
+import react from "@vitejs/plugin-react";
+import { defineConfig } from "vite";
+import { nodePolyfills } from "vite-plugin-node-polyfills";
+import svgr from "vite-plugin-svgr";
+
+// eslint-disable-next-line import/no-unused-modules
+export default defineConfig({
+  base: "./",
+  plugins: [
+    react(),
+    svgr(),
+    nodePolyfills({
+      include: ["stream", "util", "crypto", "vm"],
+      globals: {
+        Buffer: true,
+      },
+    }),
+  ],
+  optimizeDeps: {
+    esbuildOptions: {
+      // Node.js global to browser globalThis
+      define: {
+        global: "globalThis",
+      },
+    },
+  },
+  build: {
+    rollupOptions: {
+      input: {
+        embed: resolve(import.meta.dirname, "index.html"),
```

Comment:
is this needed? the root/index.html is used by default anyway

Reply:
Good point - removed now. Thank you!